<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QWindow</class>
 <widget class="QMainWindow" name="QWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1490</width>
    <height>880</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>1490</width>
    <height>880</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1490</width>
    <height>880</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>DICOM_VIEWER</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>1450</width>
     <height>700</height>
    </size>
   </property>
   <widget class="QSplitter" name="splitter">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>10</y>
      <width>1471</width>
      <height>761</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <widget class="QListWidget" name="listWidget">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>250</width>
       <height>715</height>
      </size>
     </property>
    </widget>
    <widget class="QTabWidget" name="tabWidget">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>735</width>
       <height>720</height>
      </size>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab_1">
      <attribute name="title">
       <string>2D view</string>
      </attribute>
      <widget class="QVTKOpenGLWidget" name="view1" native="true">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>710</width>
         <height>710</height>
        </rect>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
         <horstretch>200</horstretch>
         <verstretch>200</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>710</width>
         <height>710</height>
        </size>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_2">
      <property name="minimumSize">
       <size>
        <width>740</width>
        <height>740</height>
       </size>
      </property>
      <attribute name="title">
       <string>3D view</string>
      </attribute>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>709</width>
         <height>711</height>
        </rect>
       </property>
       <layout class="QGridLayout" name="gridLayout">
        <item row="1" column="0">
         <widget class="QVTKOpenGLWidget" name="view2_3" native="true">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
            <horstretch>200</horstretch>
            <verstretch>200</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>350</width>
            <height>350</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QVTKOpenGLWidget" name="view2_2" native="true">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
            <horstretch>200</horstretch>
            <verstretch>200</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>350</width>
            <height>350</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QVTKOpenGLWidget" name="view2_4" native="true">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
            <horstretch>200</horstretch>
            <verstretch>200</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>350</width>
            <height>350</height>
           </size>
          </property>
          <property name="baseSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QVTKOpenGLWidget" name="view2_1" native="true">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
            <horstretch>200</horstretch>
            <verstretch>200</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>350</width>
            <height>350</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </widget>
    <widget class="QTableWidget" name="tableWidget">
     <property name="minimumSize">
      <size>
       <width>450</width>
       <height>715</height>
      </size>
     </property>
     <column>
      <property name="text">
       <string>(Group, Element)</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Tag description</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Value</string>
      </property>
     </column>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1490</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionOpenDicomDir"/>
    <addaction name="actionOpen_Zip_file"/>
    <addaction name="actionScan_for_Dicom_file"/>
    <addaction name="separator"/>
    <addaction name="actionPrint_Setup"/>
    <addaction name="separator"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="actionToolBar"/>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>Tools</string>
    </property>
    <addaction name="actionCustomize"/>
    <addaction name="actionOptions"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout_Dicom_Viewer"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuView"/>
   <addaction name="menuTools"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionOpen"/>
   <addaction name="actionOpenDicomDir"/>
   <addaction name="separator"/>
   <addaction name="actionAbout_Dicom_Viewer"/>
   <addaction name="separator"/>
   <addaction name="actionExit"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionAbout_Dicom_Viewer">
   <property name="icon">
    <iconset>
     <normaloff>Icons/info1.png</normaloff>Icons/info1.png</iconset>
   </property>
   <property name="text">
    <string>About Dicom_Viewer</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="icon">
    <iconset>
     <normaloff>Icons/folder1.png</normaloff>Icons/folder1.png</iconset>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionOpenDicomDir">
   <property name="icon">
    <iconset>
     <normaloff>Icons/directory1.png</normaloff>Icons/directory1.png</iconset>
   </property>
   <property name="text">
    <string>Open DicomDir...</string>
   </property>
  </action>
  <action name="actionOpen_Zip_file">
   <property name="icon">
    <iconset>
     <normaloff>Icons/Zip file.png</normaloff>Icons/Zip file.png</iconset>
   </property>
   <property name="text">
    <string>Open Zip file...</string>
   </property>
  </action>
  <action name="actionScan_for_Dicom_file">
   <property name="text">
    <string>Scan for Dicom file...</string>
   </property>
  </action>
  <action name="actionPrint_Setup">
   <property name="text">
    <string>Print Setup</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="icon">
    <iconset>
     <normaloff>Icons/Exit.png</normaloff>Icons/Exit.png</iconset>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionToolBar">
   <property name="text">
    <string>ToolBar</string>
   </property>
  </action>
  <action name="actionCustomize">
   <property name="text">
    <string>Customize</string>
   </property>
  </action>
  <action name="actionOptions">
   <property name="text">
    <string>Options...</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QVTKOpenGLWidget</class>
   <extends>QWidget</extends>
   <header>QVTKOpenGLWidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
